man/setDT.Rd

```diff
@@ -58,6 +59,23 @@ setDT(X, key="a")[]
 X = list(a=1:5, a=6:10)
 setDT(X, check.names=TRUE)[]
 
+# FAQ: Dealing with data.table loaded from RDS or RData files
+# Example to demonstrate the usage of setDT after loading from RDS
+# Step 1: Define file path
```

Comment:
I don't think we need to enumerate every trivial step. Example readers are expected to know R basics. We only need to comment on things specific to {data.table}.